### PR TITLE
[OFFAPPS-673] updated translations

### PR DESCRIPTION
**Pull request by pmgrove**
:v:

Updated translations

/cc @zendesk/vegemite @ydrain @injuhwang 

### References
 - Jira link: https://zendesk.atlassian.net/browse/OFFAPPS-673

### Risks
 - None

### CODE REVIEW

File: translations/ar.json [hunk 1]
```diff
@@ -71,14 +71,14 @@
     },
     "condition": {
       "equal": "=",
-      "greater": "&gt;",
-      "less": ""
+      "greater": ">",
+      "less": "&lt;"
```
Comment (adammw) [inline translations/ar.json]: This translation looks incorrect looking at English. Strange considering greater was corrected... Does it still display correctly?

**Comment by sandlerr** [comment]
this is weird but the usage is in `{{ t ...condition.greater }}` so it will be escaped again. If it was `{{{ t ...condition.greater }}}` then there might be a problem?

**Comment by pmgrove** [comment]
@sandlerr, I am confused, do we need to do something here?

**Comment by sandlerr** [comment]
:+1:

**Comment by pmgrove** [comment]
@ydrain, @injuhwang why has `greater` changed from using `&gt;` to `>` but less is still using `&lt;` ?

**Comment by ydrain** [comment]
@craig-day `txt.apps.search_app.search.condition.less` & `txt.apps.search_app.search.condition.greater` are marked as `hidden`, and Arabic is falling back to English. Here are the value that Rosetta returning:

`"txt.apps.search_app.search.condition.greater":"\u003e"`
`"txt.apps.search_app.search.condition.less":"\u0026lt;"`
https://support.zendesk.com/api/v2/rosetta/locales/en.json?include=translations&packages=app_search_app

Why the less than string is not "\u003c"?

**Comment by sandlerr** [comment]
in fact the html entities are possibly better here, angle brackets have special meaning in HTML pages but &lt; and &gt; should display as less than and greater than signs respectively

**Comment by craig-day** [comment]
So I _think_ what is happening here, is that the `>` sign by itself doesn't mean anything special.  The `<` on the other hand means the start of an HTML tag.  When Rosetta serves translations, it calls `sanitize` on them by default.  So if we look at the strings in English, the `>` gets converted to `&gt;` as it should, but the `<` seems to get over sanitized and converted to `&amp;lt;`.  I'm guess this is due to sanitization being run twice, once by Rosetta and once by whatever is rendering the page.  You can tell Rosetta to not sanitize by adding the query parameters `&raw=true` if you would like.

**Comment by sandlerr** [comment]
> the `<` seems to get over sanitized and converted to `&amp;lt;`

is that a bug?

**Comment by craig-day** [comment]
@sandlerr the double sanitize is. I'll try to look at it in the next day or two.

**Comment by ydrain** [comment]
Created a JIRA ticket: https://zendesk.atlassian.net/browse/LOCAL-836

**Comment by pmgrove** [comment]
@ydrain, I have checked this on Staging and all is working as expected even with the current translations for `greater` and `less`. Are you happy for this to be deployed and then we can update the translations again when they have been changed?

**Comment by ydrain** [comment]
If the functioning ok, please merge as is. You can remove the translations if you wish. :+1: 

We will follow up the behavior with https://zendesk.atlassian.net/browse/LOCAL-836. Thanks!

**Pull request merged**